src/editor/ui_menus/main_menu_map_options.cc

```diff
@@ -349,16 +354,18 @@ MainMenuMapOptions::MainMenuMapOptions(EditorInteractive& parent, Registry& regi
 	teams_box_.add(new UI::Textarea(
 	   &teams_box_, 0, 0, max_w_, labelh_,
 	   (boost::format(ngettext("%u Player", "%u Players", nr_players)) % nr_players).str()));
-	teams_box_.add_space(4);
+	teams_box_.add_space(padding_);
 	teams_box_.add(new UI::Textarea(&teams_box_, 0, 0, max_w_, labelh_, _("Suggested Teams:")));
+	teams_box_.add_space(padding_);
 	teams_box_.add(&inner_teams_box_, UI::Box::Resizing::kFullSize);
+	teams_box_.add_space(padding_);
 	teams_box_.add(&new_suggested_team_, UI::Box::Resizing::kFullSize);
 	new_suggested_team_.sigclicked.connect([this]() {
 		SuggestedTeamsEntry* ste =
 		   new SuggestedTeamsEntry(this, &inner_teams_box_, eia().egbase().map(),
 		                           max_w_ - UI::Scrollbar::kSize, Widelands::SuggestedTeamLineup());
 		inner_teams_box_.add(ste);
-		inner_teams_box_.add_space(kSuggestedTeamsUnitSize);
+		//		inner_teams_box_.add_space(kSuggestedTeamsUnitSize);
```

Comment:
Remove this line, then the branch is good to go :)

Reply:
?
I already removed it yesterday... ;)

Reply:
I was looking at the wrong view then.